webapp-django/crashstats/profile/jinja2/profile/profile.html

```diff
@@ -34,7 +34,7 @@ <h2>Your Profile</h2>
             </ul>
 
             <!-- clear after two floats -->
-            <div style="clear: both;"></div>
+            <div class="clear"></div>
```

Comment:
Could we use something like [this clearfix](https://css-tricks.com/snippets/css/clear-fix/) to have the container clear the links and remove the need for this element altogether?